Packs/HelloWorld/Integrations/HelloWorldX2/README.md

```diff
@@ -0,0 +1,1358 @@
+This is the Hello World integration for getting started and learn how to build an integration with Cortex XSOAR.
```

Comment:
How will content-docs work with this integration?
Will it colid with the original one since they have the same ID?

Reply:
Good thinking, not sure what the behavior would be.
Maybe we should change the ID just in case